node_agent/main.go

```diff
@@ -35,8 +35,10 @@ func init() {
 		"Node identity private key file")
 	naConfig.IstioCAAddress = flag.String("ca-address", "127.0.0.1",
 		"Istio CA address")
-	naConfig.PercentageExpirationTime = flag.Int("renew-wait-time", 60,
-		"Percentage time to elapse before renewing certificate")
+	naConfig.CertRenewalGracePeriodPercentage = flag.Int("renew-grace-period-percent",
```

Comment:
I meant picking a default as a constant, instead of making it configurable. Flexibility could lead to confusion and I don't think anyone really wants to configure this value so exposing an option may clog the CLIs.

Anyway I don't feel super strong about this so it's up to you.